CRM/Upgrade/Incremental/sql/5.63.alpha1.mysql.tpl

```diff
@@ -14,3 +14,19 @@ VALUES
     ( {$domainID}, 'https://civicrm.org/help?src=iam',       '{ts escape="sql" skip="true"}Get Help{/ts}',   'Get Help',   NULL, 'AND', @adminHelplastID, '1', NULL, 2 );
 
 UPDATE IGNORE `civicrm_navigation` SET `name` = 'Register Your Site', `label` = '{ts escape="sql" skip="true"}Register Your Site{/ts}' WHERE `name` = 'Register your site';
+
+-- Ensure new name field is not null/unique. Setting to ID is a bit lazy - but sql localisation is painful.
```

Comment:
@eileenmcnaughton - is this an outdated comment? You seem to have pushed through the pain.

Reply:
no - that update is still just setting name to id